rviz_rendering/src/rviz_rendering/ogre_render_window_impl.cpp

```diff
@@ -477,16 +472,15 @@ void RenderWindowImpl::setBackgroundColor(Ogre::ColourValue background_color)
 
 void RenderWindowImpl::setCameraAspectRatio()
 {
-  // auto width = parent_->width();
-  auto width = parent_->width() ? parent_->width() : 100;
-  // auto height = parent_->height();
-  auto height = parent_->height() ? parent_->height() : 100;
+  auto width = parent_->width();
+  // auto width = ogre_render_window_->getWidth() ? ogre_render_window_->getWidth() : 100;
+  auto height = parent_->height();
+  // auto height = ogre_render_window_->getHeight() ? ogre_render_window_->getHeight() : 100;
```

Comment:
Maybe we can just remove these commented lines? I'm not sure why they were there to begin with.